frontend/src/css/Sidebar.styl

```diff
@@ -7,11 +7,13 @@ sidebarColor = #333
 sidebarColorDarken = darken(sidebarColor, 15%)
 
 .sidebar
+  position relative
   &__expand
-    position fixed
     left sidebar-width
+    top 0
+    position absolute
     display flex
-    z-index 10
+    z-index 100
```

Comment:
Why change the z-index? It's bigger then a flash message.
(ref https://github.com/tracim/tracim/issues/2840#issuecomment-645277067)

Reply:
It's because the scollview was partially hiding the button. 15 should be enough. Is this okay for you?

Reply:
ok for me

Reply:
```suggestion
    z-index 15
```